aiqc/__init__.py

```diff
@@ -985,7 +1008,28 @@ def to_pandas(
 			columns:list = None, 
 			samples:list = None
 		):
-			return Dataset.Tabular.to_pandas(id, columns, samples)
+			df = Dataset.Tabular.to_pandas(id, columns, samples)
+			word_counts, feature_names = Dataset.Text.get_feature_matrix(df)
+			df = pd.DataFrame(word_counts.todense(), columns = feature_names)
+			return df
+
+		
+		def to_numpy(
+			id:int, 
+			columns:list = None, 
+			samples:list = None
+		):
+			df = Dataset.Tabular.to_pandas(id, columns, samples)
+			word_counts, feature_names = Dataset.Text.get_feature_matrix(df)
+			return word_counts, feature_names
+
+
+		def get_feature_matrix(
+			dataframe:object
+		):
+			count_vect = CountVectorizer()
+			word_counts = count_vect.fit_transform(dataframe[Dataset.Text.column_name].tolist())
```

Comment:
if you `fit` then `transform` separately, then you can later use that `fit` object to `inverse_transform` the numbers back to text. Useful when making predictions and for human review.

These are actually pretty similar to the sklearn.preprocessing methods, so maybe they can go in Encoderset in the future. But keep charging down this path for now!

Reply:
so serialize the fit object and persist it in the database?

I agree, encoderset also seems like a good place (probably better) for this transformation as nothing useful is happening between dataset step and encoderset step by performing this transformation